_blog.yml

```
@@ -2232,3 +2232,11 @@
     - aws
     - partnership
     - guide
+
+- local: cnil-accompaniment
```

```suggestion
- local: cnil
```

either just `cnil` or a more general `data-protection-at-hf` or `eu-data-protection`?

(accompaniement seems off to me)